macros/create_regex_udf.sql

```diff
@@ -0,0 +1,14 @@
+{% macro create_regexp_replace_udf() %}
```

Comment:
thanks @NiallRees ! should we add a bit to readme on:

1. how to run
2. required permissions


?

Reply:
Deliberately left off for now - let's see if anyone runs into permissions issues. The user doesn't need to do anything, it runs automatically.